code/api/package.json

```diff
@@ -12,7 +12,11 @@
     "start": "nodemon src/index.js --exec babel-node",
     "start:prod": "npm run build:prod && npm run start:server",
     "build:prod": "babel src -s -D -d build",
-    "start:server": "node build/index.js"
+    "start:server": "node build/index.js",
+    "test": "jest"
+  },
+  "jest": {
+    "setupFilesAfterEnv": ["jest-extended"]
   },
```

Comment:
Added this extended package for Jest both here and on the command line within this directory so that I could use matchers like `expect().toBeNumber` and `expect().toBeNull`